appshell/appshell_extensions.js

```diff
@@ -375,6 +375,42 @@ if (!brackets) {
         }, path);
     };
 
+    /**
+     * Open Document in XD App
+     *
+     * @param {string} path The Document Path to be opened.
+     * @param {function(err)} callback Asynchronous callback function. The callback gets one arguments 
+     *        (err)
+     * 
+     *        Possible error values:
+     *          NO_ERROR
+     *          ERR_INVALID_PARAMS
+     *          ERR_NOT_FOUND
+     *
+     * @return None. This is an asynchronous call that sends all return information to the callback.
+     */
+    native function OpenDocumentWithXDApp();
+    appshell.app.OpenDocumentWithXDApp = function (path, callback) {
+        var resultString = OpenDocumentWithXDApp(callback, path);
```

Comment:
We should allow use of _dummyCallback here as well

Reply:
Fixed